src/pages/overlay/components/buttons/Buttons.tsx

```diff
@@ -1,10 +1,11 @@
-import { useMemo } from 'react'
+import { useMemo, useState } from "react";
 
 import { classes } from '../../../../utils/classes'
 
 import styles from './buttons.module.scss'
+import Tooltip from "../tooltip/Tooltip";
```

Comment:
```suggestion
import { classes } from '../../../../utils/classes'
import Tooltip from '../tooltip/Tooltip'

import styles from './buttons.module.scss'
```
Nitpick, import sorting + quotes/semi consistency